x-pack/plugin/build.gradle

```diff
@@ -74,6 +76,96 @@ tasks.named("yamlRestTest").configure {
 
   // TODO: fix this rest test to not depend on a hardcoded port!
   def blacklist = ['getting_started/10_monitor_cluster_health/*']
+
+  if (BuildParams.isSnapshotBuild() == false) {
```

Comment:
This bit can probably move to a `tasks.withType(Test).configureEach { }` block since it's applicable to _all_ tests.